hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -411,15 +411,15 @@
       'pwrmgr.pwr_rst'   : ['rstmgr.pwr'],
       'pwrmgr.pwr_clk'   : ['clkmgr.pwr'],
       'flash_ctrl.keymgr': ['keymgr.flash'],
-      'alert_handler.crashdump': ['rstmgr.alert_dump']
+      'alert_handler.crashdump': ['rstmgr.alert_dump'],
+      'clkmgr.idle'      : [],
```

Comment:
leave a comment here to indicate that `clkmgr.idle` is auto populated by the merge function.